GEMORC/GEMORC-IOC-02App/Db/Makefile

```diff
@@ -0,0 +1,18 @@
+TOP=../..
```

Comment:
Doesn't make sense to have multiple IOCs for this device, it's impossible to physically have more than one ORC on the beamline. Additionally there is only one of these devices in existence (on GEM)